housing_cooperative_base/models/housing.py

```diff
@@ -17,10 +17,9 @@ class HousingPlan(models.Model):
 class Housing(models.Model):
     _name = "hc.housing"
     _description = "Housing"
+    _inherit = "hc.premise"
 
-    name = fields.Char(string="Name", required=False)
-    active = fields.Boolean(string="Active?", default=True)
-    code = fields.Char(string="code", required=True)
+    code = fields.Char(string="Code", required=True)
```

Comment:
the code could probably be useful in any premise.

Reply:
moved it!